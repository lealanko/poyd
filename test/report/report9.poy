(* Build report asciitrees default *)
read ("1.fas", "30.ss")
set(seed:50)
build (1)
report (asciitrees)