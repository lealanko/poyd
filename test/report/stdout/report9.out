Tree with cost 3253
  |--------------------------------------t1   
  |                                           
  |   |----------------------------------t28  
  |   |                                       
  |   |     |----------------------------t46  
  |   |   |-|                                 
  |   |   | |----------------------------t49  
  |   |   |                                   
  |   |   |   |--------------------------t34  
  |   |   | |-|                               
  |   |   | | |--------------------------t7   
  |   | |-| |                                 
  |   | | | | |--------------------------t4   
  |   | | |-| |                               
  |   | |   | | |------------------------t5   
  |   | |   |-| |                             
  |   | |     |-| |----------------------t2   
  |   | |       |-|                           
  |   | |         |----------------------t3   
  |   | |                                     
  |   | |     |--------------------------t11  
  |   | |     |                               
  |   | |   |-| |------------------------t12  
  |   | |   | |-|                             
  | |-| |   |   |------------------------t40  
  | | | |   |                                 
  | | | | |-| |--------------------------t47  
  | | | | | | |                               
  | | |-| | |-| |------------------------t13  
  | |   | |   |-|                             
  | |   | |     |------------------------t48  
  | |   | |                                   
  | |   | |   |--------------------------t8   
  | |   | |   |                               
  | |   | | |-| |------------------------t10  
  | |   | | | |-|                             
  | |   |-| |   |------------------------t9   
  | |     | |                                 
  | |     | | |--------------------------t17  
  | |     | | |                               
  | |     |-| | |------------------------t23  
  | |       | | |                             
  | |       |-| | |----------------------t44  
  | |         |-| |                           
  | |           |-| |--------------------t22  
  | |             |-|                         
  | |               |--------------------t24  
  | |                                         
--| |   |--------------------------------t15  
  | | |-|                                     
  | | | |--------------------------------t16  
  | | |                                       
  | | |   |------------------------------t29  
  | | | |-|                                   
  | | | | |------------------------------t35  
  | | | |                                     
  | | | | |------------------------------t45  
  | | | | |                                   
  | | | | |     |------------------------t37  
  | | | | |   |-|                             
  |-| | | |   | |------------------------t6   
    | | | |   |                               
    | | | |   | |------------------------t33  
    | | | | |-| |                             
    | | | | | | | |----------------------t14  
    | | | | | |-| |                           
    | | | | |   |-| |--------------------t38  
    | | | | |     |-|                         
    | | | | |       |--------------------t39  
    | | | | |                                 
    | | | | |   |------------------------t27  
    | | | | |   |                             
    |-| | | | |-| |----------------------t25  
      | | | | | |-|                           
      |-| | | |   |----------------------t26  
        | | | |                               
        |-| | | |------------------------t36  
          | | | |                             
          | | | | |----------------------t32  
          |-| | | |                           
            | | | |   |------------------t20  
            | | | | |-|                       
            | | | | | |------------------t31  
            | | | | |                         
            |-| | | | |------------------t18  
              | | | | |                       
              |-| | | |   |--------------t43  
                |-| | |   |                   
                  |-| | |-| |------------t21  
                    | | | |-|                 
                    |-| |   |------------t30  
                      | |                     
                      |-| |--------------t19  
                        | |                   
                        |-| |------------t41  
                          |-|                 
                            |------------t42  
